next.config.js

```diff
@@ -10,5 +10,33 @@ module.exports = withImages({
       config.externals = ['tls', 'net', 'fs'];
     }
     return config;
-  }
+  },
+  async headers() {
+    return [
+      {
+        // matching all API routes
+        source: '/api/:path*',
+        headers: [
+          { key: 'Access-Control-Allow-Credentials', value: 'true' },
+          {
+            key: 'Access-Control-Allow-Origin',
+            value: 'https://map.hap.ke.sensors.africa/',
+          },
+          {
+            key: 'Access-Control-Allow-Origin',
+            value: 'http://127.0.0.1:8080',
+          },
+          {
+            key: 'Access-Control-Allow-Methods',
+            value: 'GET,OPTIONS,PATCH,DELETE,POST,PUT',
+          },
+          {
+            key: 'Access-Control-Allow-Headers',
+            value:
+              'X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version',
+          },
+        ],
+      },
+    ];
+  },
```

Comment:
Do we still need this even with new domain?

Reply:
@kilemensi Yap, we kinda do need this to avoid cors errors on the map because we are using the dashboard url to get the nodes